### DISCUSSION TITLE
Does SPD logger supports logging to same file from different processes and is logging in this case synchronized?

### DISCUSSION THREAD

**Discussion by mushtaqk07** [gabime/spdlog#2275] [Q&A]
Hi,

I see spd logger being used in one process context. 
I have a requirement to log traces to HW access from multiple processes, in this can I use spd logger to log to file from multiple processes.

**Comment by tt4g** [comment]
Duplicate #2092 #937

spdlog does not support multi-process behavior.

There is unofficial multi-process support: https://github.com/gabime/spdlog/issues/937#issuecomment-913403352
